### ISSUE TITLE
Automatic resize and screen height

### ISSUE THREAD

**Issue by DegrangeM** [h5p/h5p-moodle-plugin#332]
For exemple, if you create a drag and drop activity with a square size.

When moodle embed the h5p content, it will make it fit the whole width by scaling it up.

As computer screeen are wider than tall, this will make the activity height be greater than the height of the screen which means the user will never be able to see all the activity without scrolling everytime (which is very problematic for a drag and drop).

Maybe an option could be added to force the h5P content to fit the screen at whole ? 

edit : must be an option because this behaviour might be unwated for exemple for the column content.

**Comment by thomasmars** [comment]
When embedding you can set the width and height on the embed frame, or if you're using the resize script with the iframe you can wrap the iframe in an element with a restricted height and width. E.g.:
```html
<div style="width:400px;">
  <iframe ..../>
</div>
```

**Comment by DegrangeM** [comment]
I am talking about the activity which doesn't allow any resize.

Currently I use generico filter to inject css wich limit height but it is not a user friendly way to do it.

**Comment by thomasmars** [comment]
Yeah, you're right, there's no easy way to do this currently. One way to solve it would be by adding additional optional fields to the mod_form for setting max width or height.

**Comment by DegrangeM** [comment]
The best things would be to be able to choose among:
- custom max width or height (for content created for a specific height or width)
- max width
- automatic : `.h5p-iframe-wrapper { max-width: 100vmin; margin:auto; }`

**Comment by MrGit66** [comment]
The same thing happens with the activity "find the hotspot" if you put an image linked to the hotspot there is no way to resize it, it will appear extremely big
